smart-contracts/test/test_ibcTokenExport.js

```diff
@@ -0,0 +1,144 @@
+const Web3Utils = require("web3-utils");
```

Comment:
Is this a brand new JS file? Why was it not made as a typescript file?

Reply:
will change it to ts file